content/tools/herman.md

```diff
@@ -97,3 +98,23 @@ and keep everything in one place.
   collections.all | getData('data.accoutrement_modules'),
   title='Integrate with Accoutrement'
 ) }}
+
+{{ layout.title('OSS Sponsors') }}
+[comment]: <> (## Great Horned Owl)
+
+[comment]: <> (## Blue-Footed Booby)
+
+## Common Loons
```

Comment:
@SondraE I'm not sure this specific heading works here above the widget since that is the name on one level of sponsorship but the widget shows all levels.